Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -496,23 +513,48 @@ private void GetFile(DbConnection conn)
                     reader.GetString((int)SFResultSet.PutGetResponseRowTypeInfo.ResultStatus));
 
                 // Check file contents
-                using (var streamReader = new StreamReader(t_outputFilePath))
+                foreach (var line in ReadOutputFileLinesUnzippingIfReallyCompressed())
```

Comment:
that unzipping (if compressed) is a side effect of file type and autocompression switch, does it matter here so much that we have to complicate the name?

Reply:
changed to ReadOutputFileLines()